src/panels/lovelace/cards/hui-light-card.ts

```diff
@@ -0,0 +1,252 @@
+    if (changedProps.get("hass")) {
+      return (
+        changedProps.get("hass").states[this.config!.entity] !==
+        this.hass!.states[this.config!.entity]
+      );
+    }
+    return changedProps;
+  }
+
+  protected firstUpdated() {
+    jQuery("#light", this.shadowRoot).roundSlider({
+      ...lightConfig,
+      change: (value) => this._setBrightness(value),
+      drag: (value) => this._dragEvent(value),
+    });
+  }
+
+  protected updated() {
+    const attrs = this.hass!.states[this.config!.entity].attributes;
+
+    jQuery("#light", this.shadowRoot).roundSlider({
+      value: Math.round((attrs.brightness / 254) * 100) || 0,
+    });
+  }
+
+  private renderStyle() {
+    return html`
+    ${roundSliderStyle}
+    <style>
+      :host {
+        display: block;
+      }
+      ha-card {
+        overflow: hidden;
+      }
+      #root {
+        position: relative;
+        overflow: hidden;
+      }
+      #tooltip {
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        height: 100%;
+        text-align: center;
+        z-index: 15;
+      }
+      .icon-state {
+        display: block;
+        margin: auto;
+        width: 100%;
+        height: 100%;
+        transform: translate(0,25%);
+      }
+      #light {
+        margin: 0 auto;
+        padding-top: 16px;
+      }
+      #light .rs-bar.rs-transition.rs-first, .rs-bar.rs-transition.rs-second{
+        z-index: 20 !important;
+      }
+      #light .rs-range-color  {
+        background-color: var(--primary-color);
+      }
+      #light .rs-path-color  {
+          background-color: #d6d6d6;
+      }
+      #light .rs-handle  {
+          background-color: #FFF;
+          padding: 7px;
+          border: 2px solid #d6d6d6;
+      }
+      #light .rs-handle.rs-focus  {
+          border-color:var(--primary-color);
+      }
+      #light .rs-handle:after  {
+          border-color: var(--primary-color);
+          background-color: var(--primary-color);
+      }
+      #light .rs-border  {
+        border-color: transparent;
+      }
+      ha-icon {
+        display: flex;
+        margin: auto;
+        width: 40%;
+        height: 40%;
+        color: var(--paper-item-icon-color, #44739e);
+        cursor: pointer;
+      }
+      ha-icon[data-domain=light][data-state=on],
+      ha-icon[data-domain=switch][data-state=on],
+      ha-icon[data-domain=binary_sensor][data-state=on],
+      ha-icon[data-domain=fan][data-state=on],
+      ha-icon[data-domain=sun][data-state=above_horizon] {
+        color: var(--paper-item-icon-active-color, #FDD835);
+      }
+      ha-icon[data-state=unavailable] {
+        color: var(--state-icon-unavailable-color);
+      }
+      .name {
+        font-size: 1.2rem;
+        padding-top: 16px;
+      }
+      .brightness {
+        1.2rem;
+      }
+    </style>
+    `;
+  }
+
+  private _dragEvent(e) {
+    this.shadowRoot!.querySelector(".brightness")!.innerHTML = e.value + "%";
```

Comment:
`TypeError: Cannot read property 'nodeType' of null` 

I get the above error when dragging the brightness

Reply:
It means `.brightness` cannot be found inside the shadow root.

Reply:
Or `this.shadowRoot` is null, which seems odd.

Reply:
Actually, I miss spoke. I when I let go to change change the brightness. After something changes The icon doesnt change brightness and I get that error